src/pcapi/models/feature.py

```diff
@@ -98,3 +98,14 @@ def remove_feature_from_database(feature: FeatureToggle) -> None:
     """
     statement = text("DELETE FROM feature WHERE name = :name").bindparams(name=feature.name)
     op.execute(statement)
+
+
+def get_features_dict(feature_names: list[FeatureToggle]) -> dict:
+    features = Feature.query.filter(Feature.name.in_(feature_names)).with_entities(Feature.name, Feature.isActive).all()
+
+    feature_dict = {}
+
+    for feature in features:
+        feature_dict[feature[0]] = feature[1]
```

Comment:
Suggestion :

```python
# from sqlalchemy.orm import load_only

features = (
    Feature.query.filter(Feature.name.in_(feature_names)).options(load_only(Feature.name, Feature.isActive)).all()
)

feature_dict = {}

for feature in features:
    feature_dict[feature.name] = feature.isActive

```

(mais c'est du détail)

Reply:
Puisqu'on en est à proposer des variantes :)

```python
features = Feature.query.filter(Feature.name.in_(feature_names)).with_entities(Feature.name, Feature.isActive).all()
return {name: is_active for name, is_active in features}
```


(En vrai, je dis ça juste pour participer au concours, le choix final m'importe peu. :) )

Reply:
Pourquoi faire simple quand on peut faire compliqué:
```suggestion
    return dict(Feature.query.filter(Feature.name.in_(feature_names)).with_entities(Feature.name, Feature.isActive).all())
```

Reply:
Je pense que Xavier a gagné.

"You win. Perfect!"